--- origsrc/qt3d-opensource-src-5.6.2/src/3rdparty/assimp/assimp.pri	2016-08-12 03:54:01.000000000 -0500
+++ src/qt3d-opensource-src-5.6.2/src/3rdparty/assimp/assimp.pri	2016-10-31 23:18:20.677410200 -0500
@@ -10,7 +10,7 @@ CONFIG -= precompile_header
 
 win32:DEFINES+=_CRT_SECURE_NO_WARNINGS
 
-contains(QT_CONFIG, system-zlib):!if(cross_compile:host_build) {
+contains(QT_CONFIG, system-zlib) {
     if (unix|mingw):         LIBS += -lz
     else:                    LIBS += zdll.lib
 } else {
